.github/workflows/ontobot.yml

```diff
@@ -0,0 +1,82 @@
+name: Create new pull request
+
+on:
+  workflow_dispatch:
+  issues:
+    types: [ opened, edited ]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    container: obolibrary/odkfull:v1.4.1
+    strategy:
+      matrix:
+        python-version: ["3.9"]
+        os: [ ubuntu-latest ]
+
+    steps:
+      - name: Checkout main branch
+        uses: actions/checkout@v3
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v3
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Return issue number
+        id: gh-script-issue
+        uses: actions/github-script@v6
+        with:
+          # github-token: ${{ secrets.GH_TOKEN }}
+          script: |
+            const issue_number = context.issue.number
+            const repo = context.repo.owner+"/"+context.repo.repo
+            return issue_number
+      
+      - name: Return repository name
+        id: gh-script-repo
+        uses: actions/github-script@v6
+        with:
+          # github-token: ${{ secrets.GH_TOKEN }}
+          script: |
+            const repo = context.repo.owner+"/"+context.repo.repo
+            return repo
+
+      - name: Set branch name
+        id: vars
+        run: |
+          echo "resource=src/ontology/cl-edit.owl" >> $GITHUB_ENV
+          echo "branch-name=kgcl_automation_"${{ steps.gh-script-issue.outputs.result }} >> $GITHUB_ENV
+      
+      - name: Get jar & create alias command.
+        run: wget https://github.com/gouttegd/kgcl-java/releases/download/kgcl-0.2.0/kgcl-robot-standalone-0.2.0.jar -O kgcl-robot.jar
+
+      - name: Install dependencies
+        run: |
+          pip install ontobot-change-agent
+
+      - name: Run ochange
+        id: ochange
+        run: |
+          ochange process-issue ${{ env.resource }} \
+          -r ${{ steps.gh-script-repo.outputs.result }} \
+          -n ${{ steps.gh-script-issue.outputs.result }} \
+          -g ${{ secrets.GH_TOKEN }} \
```

Comment:
Is there a particular reason for not using the default `secrets.GITHUB_TOKEN`, which is automatically set up by GitHub with each workflow?

Reply:
Honestly, we did not notice another existing token. Must have been a foresight, sorry.

Reply:
No worries. :)